data/nlu/nlu.md

```diff
@@ -56,6 +56,22 @@
 - which version of python do i need
 - Which python version should I install to run Rasa example?
 
+## intent:ask_faq_rasax
```

Comment:
ask_faq_rasax is here, why did we remove it from stories? (maybe something with merging master...)

Reply:
It shouldn't have been removed from the `faqs.md` stories. I will add it back.

Reply:
Just pushed commit that addresses this.